wallet/api/admin_isolate_key.go

```diff
@@ -82,6 +83,10 @@ func validateAdminIsolateKeyParams(rawParams jsonrpc.Params) (AdminIsolateKeyPar
 		return AdminIsolateKeyParams{}, ErrPassphraseIsRequired
 	}
 
+	if params.IsolatedWalletPassphrase == "" {
+		return AdminIsolateKeyParams{}, ErrPassphraseIsRequired
```

Comment:
This is not the right error, it should be `ErrIsolatedWalletPassphraseIsRequired`